src/data/dataApi.ts

```diff
@@ -542,6 +542,35 @@ export const getChats = async (token: string | undefined) => {
   }
 }
 
+export const postDevice = async (key: string, auth: string, endpoint: string, token: string) => {
+  console.log(key && auth && endpoint && token);
+  if (key && auth && endpoint && token) {
```

Comment:
You don't need this line since all of your parameters are not nullable. So they will be defined.

- [ ] remove check.